src/ansys/dpf/core/operator_specification.py

```diff
@@ -272,7 +273,7 @@ def __str__(self):
         return "Description:\n" + str(self.description) + "\nProperties:\n" + str(self.properties)
 
     @property
-    def properties(self):
+    def properties(self) -> dict:
         """Returns some additional properties of the Operator, like the category, the exposure,
         the scripting and user names and the plugin
```

Comment:
```suggestion
        """some additional properties of the Operator, like the category, the exposure,
        the scripting and user names, and the plugin
```
Properties should be described as nouns. We also use the Oxford comma, which is placing a comma before the "and" or the "or" in a list or more than two items.